### PR TITLE
docs: update chatwoot VDP guidelines

### PR DESCRIPTION
**Pull request by vishnu-narayanan**
# Pull Request Template

## Description

Updates to chatwoot Vulnerability Disclosure Program guidelines.
- Define out of scope issues

## Type of change

Please delete options that are not relevant.
- [x] Doc update

## How Has This Been Tested?
n/a

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,8 +1,31 @@
...
 We use [huntr.dev](https://huntr.dev/) for security issues that affect our project. If you believe you have found a vulnerability, please disclose it via this [form](https://huntr.dev/bounties/disclose). 
+
 This will enable us to review the vulnerability, fix it promptly, and reward you for your efforts.
 
 If you have any questions about the process, feel free to reach out to hello@chatwoot.com.
+
+
+## Out of scope
+
+Please do not peform testing against Chatwoot production services. Use our staging environment at staging.chatwoot.com.
```
Comment (sojan-official) [inline SECURITY.md:15]: let advise to spin up a self hosted instance for now. lets set up a dedicated security testing instance later.

**Review by pranavrajs — DISMISSED** [review]

File: SECURITY.md [hunk 2]
```diff
@@ -1,8 +1,31 @@
...
+
+- Missing HTTP security headers
+- Self XSS
+- HTTP Host Header XSS without working proof-of-concept
+- Incomplete/Missing SPF/DKIM
+- Denial of Service attacks
+- DNSSEC
+- Social Engineering attacks
+
+If you are not sure about the scope, please create a report.
```
Comment (sojan-official) [inline SECURITY.md:27]: let's mention the hello@chatwoot.com email here. Since huntr isn't suited for reporting things related to infra etc.

Comment (vishnu-narayanan) [reply SECURITY.md:27]: @sojan-official can we have `security@chatwoot.com` for all things security ?

Comment (sojan-official) [reply SECURITY.md:27]: @vishnu-narayanan added `security@chatwoot.com`

**Review by pranavrajs — DISMISSED** [review]

**Pull request merged**